tests/qmltests/Stage/tst_PhoneStage.qml

```diff
@@ -251,10 +251,11 @@ Item {
             var appsToTest = 6;
             var apps = new Array();
             for (var i = 0; i < appsToTest; i++) {
-                var item = new Object();
-                item.tag = "App " + i;
-                item.index = i;
-                item.total = appsToTest;
+                var item = {
+                    tag: "App " + i,
+                    index: i,
+                    total: appsToTest
+                };
```

Comment:
Although I actually prefer this, why does the old code cause an error?

Reply:
I think it was related to the `Object` issues from uitk, as there was a `var item = new Object();` instead.

Reply:
Right.